src/cashweb/auth_wrapper/wrapper_pb.d.ts

```diff
@@ -1,114 +1,148 @@
 // package: wrapper
 // file: wrapper.proto
 
-import * as jspb from "google-protobuf";
+import * as jspb from 'google-protobuf'
```

Comment:
These should have been excluded from the linter entirely as they're generated files.